packages/manager/modules/pci/src/projects/new/payment/components/default/index.html

```diff
@@ -4,27 +4,56 @@
         data-translate="pci_project_new_payment_default_title"
     ></h2>
 
-    <p
-        data-translate="pci_project_new_payment_default_explain"
-        data-translate-values="{ paymentMehodSection: $ctrl.sectionUrls.payment, myAccountSection: $ctrl.sectionUrls.myAccount }"
+    <p class="text-justify"
+       data-translate="pci_project_new_payment_default_explain_info"
     ></p>
 
+
+    <!--Default credit card -->
     <div class="default-method">
-        <img
-            class="d-inline-block vertical-middle"
-            data-ng-if="$ctrl.defaultPaymentMethod.icon.data"
-            data-ng-src="{{ $ctrl.defaultPaymentMethod.icon.data }}"
-            alt=""
-        />
-        <span
-            class="d-inline-block vertical-middle"
-            data-ng-bind="$ctrl.defaultPaymentMethod.label"
-        >
-        </span>
-        <strong
-            class="text-success d-inline-block vertical-middle"
-            data-translate="pci_project_new_payment_default_method"
-        >
-        </strong>
+
+        <!--Go to payment section-->
+        <p class="text-right">
+            <a data-ng-href="{{::$ctrl.sectionUrls.payment}}"
+               target="_blank"
```

Comment:
No need for a rel=noopener ?

Reply:
it's redirect to our Manager.
Do you think it's a security leaks ?